comp/core/workloadmeta/collectors/util/ecs_util_test.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build docker
+
+package util
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseRegionAndAWSAccountID(t *testing.T) {
```

Comment:
Should we add tests for the other function of the `ecs_util` package?

Reply:
I think it's not necessary as we already have them in
https://github.com/DataDog/datadog-agent/blob/882d6e4354906f23c0124aa0d908eed0e5bcbd4a/comp/core/workloadmeta/collectors/internal/ecsfargate/ecsfargate_test.go#L31

and https://github.com/DataDog/datadog-agent/blob/882d6e4354906f23c0124aa0d908eed0e5bcbd4a/comp/core/workloadmeta/collectors/internal/ecs/ecs_test.go#L145